adal/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,6 @@
     <application>
         <activity android:name="com.microsoft.aad.adal.AuthenticationActivity" >
         </activity>
-
```

Comment:
nit: remove this file from PR

Reply:
this is removing the redundant empty line in last PR as you commented.